vuu-ui/packages/vuu-shell/src/layout-management/useLayoutManager.tsx

```diff
@@ -1,42 +1,41 @@
 import React, { useState, useCallback, useContext, useEffect } from "react";
-import { getLocalEntity, saveLocalEntity } from "@finos/vuu-filters";
-import { LayoutJSON } from "@finos/vuu-layout";
-import { getUniqueId } from "@finos/vuu-utils";
+import { getLocalEntity } from "@finos/vuu-filters";
+import { LayoutJSON, LayoutPersistenceManager } from "@finos/vuu-layout";
 import { LayoutMetadata, Layout } from "./layoutTypes";
 
 export const LayoutManagementContext = React.createContext<{
   layouts: Layout[],
   saveLayout: (n: Omit<LayoutMetadata, "id">) => void
 }>({ layouts: [], saveLayout: () => { } })
 
-export const LayoutManagementProvider = (props: { children: JSX.Element | JSX.Element[] }) => {
-
-  const [layouts, setLayouts] = useState<Layout[]>([])
+export const LayoutManagementProvider = (props: {
+    persistenceManager: LayoutPersistenceManager,
```

Comment:
I put a similar comment on #39 - I think I lean towards the persistence manager being instantiated in this file rather than passed as a prop. For example, an environment variable which defines how we want to persist layout, and based on the value of that envar, instantiate the appropriate persistence manager. It would be easier to change configuration without changing the code. This would be a pro when we're thinking about deployment to different environments, which we're not here 😄. But I can see arguments for giving the provider the exact persistence manager you want to use to be fair. What's your thoughts?

Reply:
Although now I think about it more, the benefit of having it instantiated based on code (rather than envars) and passed as a prop is that different stories can then use different persistence managers. Like `NewThemeLocal.examples.tsx` and `NewThemeRemote.examples.tsx`. If it was purely based on envars, I'm not sure how we'd override the envar in the code for each story.

Reply:
Explore envars & initialising in LayoutProvider.

Reply:
> Explore envars & initialising in LayoutProvider as separate ticket.

Do we still want this? I feel like I had a lot to say about it on Friday but I can't remember what any of that is now.

Reply:
We added this feature to #27.